osclib/stagingapi.py

```diff
@@ -585,12 +585,14 @@ def get_open_requests(self, query_extra=None):
         # expect Request objects
 
         requests = []
+        target = f"target[@project='{self.project}']"
 
         # xpath query, using the -m, -r, -s options
-        where = f"@by_group='{self.cstaging_group}' and @state='new'"
-        target = f"target[@project='{self.project}']"
+        if not match_filter:
+            review_where = f"review[@by_group='{self.cstaging_group}' and @state='new']"
+            match_filter = f"state/@name='review' and {review_where}"
```

Comment:
I think we can fold `review_where` inside `match_filter`, but I don't have a strong opinion on that (and even previously it was split in `where` and `target` indeed).

Reply:
Thank you, I agree less bytes it is always better. Should be fixed now :)